wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationSystemMessageCell.swift

```diff
@@ -91,6 +91,43 @@ extension ConversationStartedSystemMessageCell {
 
 }
 
+class ConversationEncryptionInfoCell: ConversationIconBasedCell,ConversationMessageCell {
+    struct Configuration {}
+
+    private let encryptionLabel = DynamicFontLabel(fontSpec: .mediumRegularFont,
+                                                   color: SemanticColors.Label.textDefault)
+    private let sensitiveInfoLabel = DynamicFontLabel(fontSpec: .mediumRegularFont,
+                                                      color: SemanticColors.Label.textDefault)
+
+    func configure(with object: Configuration, animated: Bool) {}
+
+    override func configureSubviews() {
+        super.configureSubviews()
+
+        typealias connectionView = L10n.Localizable.Conversation.ConnectionView
+        encryptionLabel.numberOfLines = 0
+        encryptionLabel.text = connectionView.encryptionInfo
+        encryptionLabel.translatesAutoresizingMaskIntoConstraints = false
+        topContentView.addSubview(encryptionLabel)
+        sensitiveInfoLabel.numberOfLines = 0
+        sensitiveInfoLabel.text = connectionView.sensitiveInformationWarning
+        sensitiveInfoLabel.translatesAutoresizingMaskIntoConstraints = false
+        bottomContentView.addSubview(sensitiveInfoLabel)
+
+        lineView.isHidden = true
+        imageView.tintColor = SemanticColors.Icon.backgroundDefault
+        imageView.image =  UIImage(named: "Info")
```

Comment:
We use SwiftGen for icons: Asset.Images...

Reply:
done